lib/presentation/widgets/input_field_widget.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:flutter/material.dart';
+import 'package:water_tracker/custom_theme.dart';
+import 'package:water_tracker/form_validators.dart';
+
+class InputFieldWidget extends StatelessWidget {
+  const InputFieldWidget({
+    Key? key,
+    required this.labelText,
+    this.validator,
+  }) : super(key: key);
+
+  final String labelText;
+  final ValidatorFunction? validator;
+
+  @override
+  Widget build(BuildContext context) {
+    return TextFormField(
+      decoration: InputDecoration(
+        filled: true,
```

Comment:
but u already have it in theme, why override it here?

Reply:
I will delete